Documentation/teaching/labs/arm_kernel_development.rst

```diff
@@ -220,43 +222,43 @@ Here is an example:
 
 .. code:: c
 
-   / {
-        node@0 {
-             empty-property;
-             string-property = "string value";
-             string-list-property = "string value 1", "string value 2";
-             int-list-property = <value1 value2>;
-
-             child-node@0 {
-                     child-empty-property;
-                     child-string-property = "string value";
-                     child-node-reference = <&child-node1>;
-             };
-
-             child-node1: child-node@1 {
-                     child-empty-property;
-                     child-string-property = "string value";
-             };
-      };
-   };
-
-Qemu
+    / {
+         node@0 {
+              empty-property;
+              string-property = "string value";
+              string-list-property = "string value 1", "string value 2";
+              int-list-property = <value1 value2>;
+
+              child-node@0 {
+                      child-empty-property;
+                      child-string-property = "string value";
+                      child-node-reference = <&child-node1>;
+              };
+
+              child-node1: child-node@1 {
+                      child-empty-property;
+                      child-string-property = "string value";
+              };
+       };
+    };
+
+QEMU
 ====
 
 We will use ``qemu-system-arm`` to boot 32bit ARM platforms. Although, this can be installed from official distro repos, for example:
 
 .. code:: bash
 
-   sudo apt-get install -y qemu-system-arm
+    sudo apt-get install -y qemu-system-arm
 
 We strongly recommend using latest version of ``qemu-system-arm`` build from sources:
 
 .. code:: bash
 
-   $ git clone https://gitlab.com/qemu-project/qemu.git
-   $ ./configure --target-list=arm-softmmu --disable-docs
-   $ make -j8
-   $ ./build/qemu-system-arm
+    $ git clone https://gitlab.com/qemu-project/qemu.git
+    $ ./configure --target-list=arm-softmmu --disable-docs
```

Comment:
Maybe make it clear that this must be done from the qemu directory after cloning.